mods/default/functions.lua

```diff
@@ -406,6 +406,51 @@ function default.register_fence_rail(name, def)
 	minetest.register_node(name, def)
 end
 
+--
+-- Mese post registration helper
+--
+
+function default.register_mesepost(name, def)
+	minetest.register_craft({
+		output = name .. " 4",
+		recipe = {
+			{'', 'default:glass', ''},
+			{'default:mese_crystal', 'default:mese_crystal', 'default:mese_crystal'},
+			{' ', def.material, ''},
+		}
+	})
+
+	local post_texture = def.texture .. "^default_mese_post_light_side.png^[makealpha:255,126,126"
+	local post_texture_dark = def.texture .. "^default_mese_post_light_side_dark.png^[makealpha:255,126,126"
```

Comment:
I think this color (`255,126,126`) is risky - it could be contained in a texture pack and would be changed to alpha incorrectly. What do you think about pitch black (`0,0,0`)?

Reply:
Hmm, good idea.

Reply:
Fences still use `255,126,126`.

Reply:
Too bad. Now there are two colors that cannot be used in a wood plank texture. I'm convinced 0,0,0 is better than an arbitrary color within the brightness range of wood textures.